sdk/ai/azure-ai-projects/samples/connections/async_samples/sample_connections_async.py

```diff
@@ -121,22 +123,22 @@ async def sample_connections_async():
             print("====> Creating ChatCompletionsClient using API key authentication")
             from azure.core.credentials import AzureKeyCredential
 
-            client = ChatCompletionsClient(
-                endpoint=connection.endpoint_url, credential=AzureKeyCredential(connection.key)
+            ai_client = ChatCompletionsClient(
```

Comment:
In the sync sample version you call it `inference_client`. So if you want to change the name, you can change it to that.

Reply:
Mypy defines the variable type y its first occurrence in the code and once it was assigned specific type, it will complain if we call methods of the other type.